doc/features/metrics/README.md

```diff
@@ -0,0 +1,119 @@
+# Metrics
+
+The driver exposes measurements of its internal behavior through an API composed by abstractions. The client application must provide an implementation of these abstractions (`IDriverMetricsProvider`) in order to enable metrics.
+
+DataStax offers an [Extension package] based on the [App.Metrics] library. Application developers can use [App.Metrics] to export metrics to a monitoring tool.
```

Comment:
TIL that you can use the content of the link as the identifier.

Reply:
Oh I thought I learned that from reading your nodejs docs, maybe it was from the java driver manual I don't remember heh 😄